src/pages/workspace/members/WorkspaceMemberDetailsRoleSelectionPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {View} from 'react-native';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@navigation/types';
+import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
+import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
+import * as Policy from '@userActions/Policy';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+
+type WorkspaceMemberDetailsPageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.MEMBER_DETAILS_ROLE_SELECTION>;
+
+function WorkspaceMemberDetailsRoleSelectionPage({policyMembers, route}: WorkspaceMemberDetailsPageProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const accountID = Number(route?.params?.accountID) ?? 0;
+    const member = policyMembers?.[accountID];
+
+    const items = [
+        {
+            value: CONST.POLICY.ROLE.ADMIN,
+            text: translate('common.admin'),
+            isSelected: member?.role === CONST.POLICY.ROLE.ADMIN,
+            keyForList: CONST.POLICY.ROLE.ADMIN,
+        },
+        {
+            value: CONST.POLICY.ROLE.USER,
+            text: translate('common.member'),
+            isSelected: member?.role === CONST.POLICY.ROLE.USER,
+            keyForList: CONST.POLICY.ROLE.USER,
+        },
+    ];
+
+    const changeRole = () => {
+        if (!member) {
+            return;
+        }
+
+        const newRole = member.role === CONST.POLICY.ROLE.ADMIN ? CONST.POLICY.ROLE.USER : CONST.POLICY.ROLE.ADMIN;
+        Policy.updateWorkspaceMembersRole(route.params.policyID, [accountID], newRole);
+        Navigation.goBack();
+    };
+
+    return (
+        <ScreenWrapper testID={WorkspaceMemberDetailsRoleSelectionPage.displayName}>
+            <FullPageNotFoundView>
+                <HeaderWithBackButton
+                    title={translate('common.role')}
+                    onBackButtonPress={() => Navigation.goBack()}
+                />
+                <View style={[styles.containerWithSpaceBetween, styles.pointerEventsBoxNone]}>
+                    <SelectionList
+                        sections={[{data: items, indexOffset: 0}]}
+                        ListItem={RadioListItem}
+                        onSelectRow={changeRole}
+                        initiallyFocusedOptionKey={items.find((item) => item.isSelected)?.keyForList}
```

Comment:
I don't see the tick mark for the very first time i open a member with the new changes


https://github.com/Expensify/App/assets/46707890/1169ea0a-0d7a-4a58-aaa6-6dc84248517f

Reply:
is this a member that was invited right before? there is known issue that we need to address with it

Reply:
this is related comment: https://github.com/Expensify/App/issues/37748#issuecomment-1978908817 - we agreed to fix this while working on inviting new members